.travis.yml

```diff
@@ -36,8 +36,8 @@ env:
   - WP_VERSION=trunk WP_MULTISITE=1
 
 install:
-  - npm install
-  - export DEV_LIB_PATH=node_modules/wp-dev-lib
+  - composer install
```

Comment:
The Travis builds were failing, so I eventually moved `wp-dev-lib` to `composer`.